tests/torch/test_loggers.py

```diff
@@ -0,0 +1,111 @@
+# --------------------------------------------------------------------------------------
+# Part of the interTwin Project: https://www.intertwin.eu/
+#
+# Created by: Anna Elisa Lappe
+#
+# Credit:
+# - Anna Lappe <anna.elisa.lappe@cern.ch> - CERN
+# -------------------------------------------------------------------------------------
+
+from argparse import Namespace
+from unittest.mock import MagicMock
+
+import pytest
+from lightning.pytorch.callbacks.model_checkpoint import ModelCheckpoint
+
+from itwinai.loggers import Logger
+from itwinai.torch.loggers import PyTorchLightningLogger
+
+
+@pytest.fixture
+def lightning_loggers():
+    """Setup a generic PyTorchLightningLogger with mock loggers for testing."""
+    itwinai_logger_mock = MagicMock()
+    lightning_logger = PyTorchLightningLogger(itwinai_logger=itwinai_logger_mock)
+
+    return itwinai_logger_mock, lightning_logger
+
+
+def test_lightning_experiment_property(lightning_loggers):
+    """Test that the experiment property initializes
+    the logger context and returns the logger."""
+    itwinai_logger_mock, lightning_logger = lightning_loggers
+    experiment = lightning_logger.experiment
+    assert isinstance(experiment, Logger)
```

Comment:
Have you tested this locally? I think this will hardly pass being experiment a plain `MagicMock`

Reply:
Good catch! I have removed this function because, as jarl pointed out, it didn't add very much value, and I think what it is testing is covered sufficiently by other functions. I did add the `spec=` argument to all other MagicMocks though, that is definitely better practice!